tests/_utils/gettokenurl.js

```diff
@@ -14,27 +14,8 @@ import uid from '@ckeditor/ckeditor5-utils/src/uid';
 // into a subscription at any moment, allowing you to preserve all uploaded images.
 const CLOUD_SERVICES_TOKEN_URL = 'https://j2sns7jmy0.execute-api.eu-central-1.amazonaws.com/prod/token';
 
-export default function getToken() {
-	return new Promise( ( resolve, reject ) => {
-		const xhr = new XMLHttpRequest();
-		const userId = uid();
+export default function getTokenUrl() {
+	const userId = uid();
 
-		xhr.open( 'GET', `${ CLOUD_SERVICES_TOKEN_URL }?user.id=${ userId }` );
-
-		xhr.onload = () => {
-			if ( xhr.status >= 200 && xhr.status < 300 ) {
-				const response = JSON.parse( xhr.responseText );
-
-				resolve( response.token );
-			} else {
-				reject( new Error( `XHR status: ${ xhr.status }` ) );
-			}
-		};
-
-		xhr.onerror = err => {
-			reject( err );
-		};
-
-		xhr.send( null );
-	} );
+	return `${ CLOUD_SERVICES_TOKEN_URL }?user.id=${ userId }`;
```

Comment:
Newest CS should support anonymous users. It means that you don't need to set `user.id`, and the whole file is not needed.

Reply:
So declare `const CLOUD_SERVICES_TOKEN_URL = 'https://j2sns7jmy0.execute-api.eu-central-1.amazonaws.com/prod/token';` in every sample and snippet?

Reply:
Or

Reply:
"every sample" 

Is there more than one? :P

I would go with:
```
const CLOUD_SERVICES_TOKEN_URL = 'https://j2sns7jmy0.execute-api.eu-central-1.amazonaws.com/prod/token';

//...

cloudServices: {
	tokenUrl: CLOUD_SERVICES_TOKEN_URL
}
```

Reply:
It's used in the snippets in `ckeditor5-image`: e.g. here: https://github.com/ckeditor/ckeditor5-image/blob/master/docs/_snippets/features/image.js.

Reply:
Ach, then I would: `export CLOUD_SERVICES_TOKEN_URL`.

Reply:
I'll wait with changes for the correct token URI.